### PR TITLE
Implement new version for saving session's urlLocation

### PR DESCRIPTION
**Pull request by anna1901**
This fixes this bug:
https://trello.com/c/kqXzDa8I

### CODE REVIEW

File: AirCasting/CoreData/PersistenceController+Database.swift [hunk 1]
```diff
@@ -108,6 +108,25 @@ extension PersistenceController: SessionRemovable {
     }
 }
 
+extension PersistenceController: SessionUpdateable {
+    func updateSessionUrl(_ url: String, for sessionUUID: SessionUUID, completion: ((Error?) -> Void)?) {
+        let context = self.editContext
+        context.perform {
+            do {
+                let sessionEntity: SessionEntity = try context.newOrExisting(uuid: sessionUUID)
```
Comment (pawelgil) [inline AirCasting/CoreData/PersistenceController+Database.swift:116]: I'm wondering if we want to use `newOrExisting` here... It's clearly an error when we want to update URL of a non-existent session and by creating new session in that case we're pushing the error further down the line.

**Review by pawelgil — APPROVED** [review]

File: AirCasting/CoreData/PersistenceController+Database.swift (hunk 1 above)
Comment (anna1901) [reply AirCasting/CoreData/PersistenceController+Database.swift:116]: Good catch, that's definitely true. I'll change that.

File: AirCasting/CoreData/PersistenceController+Database.swift [hunk 2]
```diff
@@ -108,6 +108,25 @@ extension PersistenceController: SessionRemovable {
...
+            } catch {
+                Log.error("Error adding urlLocation for session: \(error)")
+                completion?(error)
+            }
+        }
+    }
+}
+
+
+
```
Comment (jankrzempek) [inline AirCasting/CoreData/PersistenceController+Database.swift:129]: Do we need that many empty lines here 😅?

**Review by jankrzempek — APPROVED** [review]
🚀

**Review by pawelgil — APPROVED** [review]

**Pull request merged**